demos/ruby-hello/ruby-hello

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
```

Comment:
@kyrofa /usr/bin/env doesn't exist in the snap dir... Is it accessing system /usr/bin/env in that case?